### PR TITLE
travis fix

### PR DESCRIPTION
**Pull request by jemten**

### CODE REVIEW

File: mip_install.pl [hunk 1]
```diff
@@ -317,6 +318,14 @@
 
 $log->info( q{Writing install instructions to:} . $SPACE . $file_name_path );
 
+## Remove selected programs from installation
+if ( $parameter{skip_program} ) {
+    foreach my $skip ( @{ $parameter{skip_program} } ) {
```
Comment (henrikstranneheim) [inline mip_install.pl]: Add loop label

File: perl_install.pl [hunk 2]
```diff
@@ -437,7 +444,7 @@ sub install_cpanm_modules {
     }
 
     ## Check aginst what's already installed
-    my @modules_to_install =
+    if my @modules_to_install =
```
Comment (henrikstranneheim) [inline perl_install.pl]: This looks odd? Why are you using an if here and without ()?

Comment (jemten) [reply perl_install.pl]: hmmm no reason to have it there whatsoever. Don't know what its doing there

**Comment by henrikstranneheim** [comment]
Great

**Pull request merged**